package.json

```diff
@@ -9,13 +9,12 @@
     "cy:open": "cypress open",
     "cy:run": "cypress run",
     "test": "npm run test:unit; npm run test:integration; npm run test:e2e",
-    "test:unit": "jest --testPathIgnorePatterns=bar",
-    "test:integration": "jest bar",
+    "test:unit": "jest --testPathIgnorePatterns=bar, chartBuilder",
+    "test:integration": "jest bar chartBuilder",
     "test:e2e": "start-server-and-test start:e2e http://localhost:8080 cy:run"
   },
   "devDependencies": {
     "@testing-library/cypress": "^10.0.1",
-    "start-server-and-test": "^2.0.3",
```

Comment:
This is needed for the e2e tests. If not included it will either cause merge conflicts or errors when running tests

Reply:
The start-server-and-test is still in the dev dependencies. Idk why, but it ended up getting arranged to be on a later line (line 26)

Reply:
Alright well that explains why the tests still passed